airbyte-cdk/java/airbyte-cdk/core/src/test/java/io/airbyte/cdk/integrations/destination_async/buffers/BufferEnqueueTest.java

```diff
@@ -19,6 +19,7 @@
 public class BufferEnqueueTest {
 
   private static final int RECORD_SIZE_20_BYTES = 20;
+  private static final String DEFAULT_NAMESPACE = "";
```

Comment:
```suggestion
  private static final String DEFAULT_NAMESPACE = "foo_namespace";
```

(and also for the other test(s))